chapters/10-community-involvement.Rmd

```diff
@@ -2,8 +2,8 @@
 
 - Designing your package to be used by or contributed to from the community
   (refer to Mozilla Science Labs Open Leaders material)
-- Contributing guidelines
 - Being inclusive (code of conduct)
+- Contributing guidelines
```

Comment:
I switched the order of contributing and code of conduct: not super important, just that code of conduct gets linked to in contributing.